lib/automation.py

```diff
@@ -203,27 +204,27 @@ def get_test_packages_url(self, properties):
                 'retry_attempts': 0,
             }
 
-            # Use hg.mo to query for the next revision which has Tinderbox builds
-            # available. We can use that to retrieve the test-packages URL.
+            # Use Treeherder to query for the next revision which has Tinderbox builds
+            # available. We can use this revision to retrieve the test-packages URL.
             if properties['tree'].startswith('release-'):
                 self.logger.info('Querying tinderbox revision for {} build...'.format(
                                  properties['tree']))
-                revision = properties['revision']
-                while True:
-                    hg_url = 'https://hg.mozilla.org/releases/{branch}/json-rev/{revision}'.format(
-                        branch=properties['branch'],
-                        revision=revision)
-                    r = requests.get(hg_url)
-                    json_data = r.json()
-
-                    # Stop once we weren't able to retrieve the revision data or
-                    # have a revision not pushed by automation.
-                    if r.status_code != 200 or json_data['user'] != 'ffxbld':
+                revision = properties['revision'][:12]
+
+                client = TreeherderClient(host='treeherder.mozilla.org', protocol='https')
```

Comment:
We are only querying for builds. Those will be the same for both. We would only need staging for us in case of submitting data. But if you feel we should be using staging here, I can read in the values from the .properties file.